vivarium/core/experiment.py

```diff
@@ -956,6 +990,15 @@ def establish_path(self, path, config, source=None):
             self.apply_config(config, source=source)
             return self
 
+    def add_inner(self, path, node):
```

Comment:
What about `add_node`? `add_inner` might be confusing since this can add to an arbitrary path, not just the current node's inner